libc/src/math/generic/scalbnf128.cpp

```diff
@@ -11,14 +11,14 @@
 #include "src/__support/FPUtil/ManipulationFunctions.h"
 #include "src/__support/common.h"
 
+#if FLT_RADIX != 2
+#error FLT_RADIX!=2, unimplemented.
```

Comment:
Consider adding a comment explaining why it is an error if this macro doesn't have this value. The error message could be restated, too, I suspect.